### DISCUSSION TITLE
open_revs=all used to return content as json array but now is multipart

### DISCUSSION THREAD

**Discussion by tudordumitriu** [apache/couchdb#3353] [Q&A] [answered]
Hi there, we are using the open_revs to get for a document all the revisions (and it has been working for a while), and as far as I see even in documentation the response should be a JSON array (https://docs.couchdb.org/en/stable/replication/conflicts.html?highlight=open_revs#working-with-conflicting-documents) but don't really know what changed lately because we are getting a multipart/mixed; content-type as response:
--58cc80049d09f221109fc4d7ad02aec6
Content-Type: application/json
{"_id":"l_64305b1cb8ed447b876b4cc2c5c5043d_org.couchdb.user:Roger","_rev":"312-d0439ff2064df8f62e2965d0017acef4","_deleted":true}
--58cc80049d09f221109fc4d7ad02aec6
Content-Type: application/json
{"_id":"l_64305b1cb8ed447b876b4cc2c5c5043d_org.couchdb.user:Roger","_rev":"415-fc9c0e1c125c53441a8108f9c5d49739"....}]}
--58cc80049d09f221109fc4d7ad02aec6--
Server version is 3.0.0

**Comment by kocolosk** [comment, marked as answer]
That's odd. CouchDB should only send a `multipart/mixed` response if the request includes an `Accept` header indicating support for that content-type:

https://github.com/apache/couchdb/blob/3.0.0/src/chttpd/src/chttpd_db.erl#L943-L968

Is it possible your client started to send a header like that?

**Reply by tudordumitriu** [reply]
Yes, that was it, thanks!
